recipes/telseq/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -x
+set -e
+
+pushd $SRC_DIR/src
+bamtools=$(conda list bamtools -c | awk -F'::' '{print $2}')
+./autogen.sh
+./configure --prefix=$PREFIX --with-bamtools=$SYS_PREFIX/pkgs/$bamtools  && make && make install
```

Comment:
This should be $PREFIX/lib or $PREFIX/include, isn't it?

Reply:
Thanks @bgruening for looking into this. lib and include subdirectories are taken care of inside the configure, which is built by autoconf with configure.ac. 

I followed the tutorial of contributing a package but unfortunately couldn't manage to build  a workflow of my own. In the end I decided to postpone this and use docker for now. This pull request can be dropped. 

Thanks.